docs/PowerAuth-SDK-for-Android.md

```diff
@@ -32,6 +32,7 @@
 - [Synchronized Time](#synchronized-time)
 - [Common SDK Tasks](#common-sdk-tasks)
 - [Additional Features](#additional-features)
+  - [Personal Information About User](#personal-information-about-user)
```

Comment:
Personal is misleading because it can be used for example only for `sub` or other custom claims that are not personal but rather IDs.

Reply:
While `sub` is still "personal information," I would standardize the terminology to "Obtaining User's Claims" (see for example [here](https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-token-claims))